apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProvider.java

```diff
@@ -377,6 +377,9 @@ public Bundle findByPatient(
                 Optional.ofNullable(serviceDate),
                 filterSamhsa,
                 includeTaxNumbers);
+      } catch (InvalidRequestException e) {
+        // If we're throwing a 400, pass it back up
+        throw e;
```

Comment:
This is important; swallowing all exceptions from here with the below Exception block was stopping me from throwing 400s and caused a lot of headaches figuring out why I wasnt getting the expected 400 once I added the code fix. Here we carve out an exception for our 400 exception so it can bubble up properly.